edx_rest_framework_extensions/auth/jwt/decoder.py

```diff
@@ -56,6 +56,7 @@ def jwt_decode_handler(token):
 
     Args:
         token (str): JWT to be decoded.
+        decode_symmetric_token (bool): Whether to decode symmetric tokens or not. Pass False for asymmetric tokens only
```

Comment:
We should note that the setting for JWT_DECODE_HANDLER expects a single argument, token. The new argument is for internal use only.